.github/workflows/ci.yml

```diff
@@ -0,0 +1,106 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+  schedule:
+  - cron: "0 0 * * *"
+
+jobs:
+  validation:
+    runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+      matrix:
+        php-version: [
+            '7.2',
```

Comment:
?

Reply:
I see that composer require >= 7.1, and old travis file supported it, should i update composer file?

Reply:
Yes.72+